composer.json

```diff
@@ -97,6 +99,8 @@
         "drupal/telephone_validation": "^2.1",
         "drupal/textarea_widget_for_text": "^1.1",
         "drupal/time_field": "^1.1",
+        "drupal/toc_filter": "^2.0@beta",
+        "drupal/toc_js": "^1.6",
```

Comment:
@radmosley, this is not a clean PR. It has a lot of dependencies from your other story. This story should only have about 12-15 files changed with reference to a required group field. Please update.

Reply:
Reset to an earlier commit and and resubmitted my commits for a cleaner PR.